app/src/main/java/org/openobservatory/ooniprobe/model/database/Measurement.java

```diff
@@ -179,6 +179,26 @@ public void deleteLogFileAfterAWeek(Context c) {
 		}
 	}
 
+	public static long getStorageUsed(Context c){
+		String database = AppDatabase.NAME + ".db";
+		return getFolderSize(getMeasurementDir(c)) +
+				c.getDatabasePath(database).length() +
+				c.getDatabasePath(database + "-shm").length() +
```

Comment:
What is the content of the `-shm` file? Should we count it?

Reply:
The WAL-Index File Format. The WAL-index or "shm" file is used to coordinate access to the database by multiple clients, and as a cache to help clients quickly locate frames within the wal file.

Reply:
👍